GOVERNANCE.md

```diff
@@ -0,0 +1,54 @@
+# Governance of daml-common
+
+## Goals
+
+`daml-common` is a set of base libraries developed in-house at DA that are meant for wide adoption across all DA products. By selecting a core set of "blessed" libraries we encourage a consistent experience for the development team as well as for the clients using our software. The types of libaries that we envisage will find their home in this repository include but are not limited to:
+
+* logging, metrics, tracing
+* resource management
+* database access abstractions
+* configuration and CLI tooling
+* TLS and access token utilities
+* self-service error codes
+* security evidencing
+* functional utilities such as non-empty monad
+
+## Process
+
+### Submission
+Inclusion of a new library in the `daml-common` repository must be proposed through a GitHub issue. A valid submission should contain
```

Comment:
You also want to have the integration into the build and release system of daml-common.

The governance doc makes the impression that all languages are welcome. This means that if we get the first submission for say Haskell, we need to extend the build system to be able to manage and release Haskell stuff. Is this intended? Or do we restrict this repo to say Scala with a single, well-defined build and release process?

Reply:
I think we restrict it to scala to start with